### PR TITLE
spreadsheet_id maybe spreadsheet_url

### PR DESCRIPTION
**Pull request by jdrzj**
Hi there,
I'm making PR for some new improvement. We want to users can apply direct link to the API. Let me know if this looks OK for you! 👋 👨‍💻

### CODE REVIEW

File: sheetsu/core/core.py [hunk 1]
```diff
@@ -19,7 +20,7 @@ class Resource(object):
     def __init__(self, client):
         """Initialize Sheetsu Resource variables
         :param client: "SheetsuClient" object instance"""
-        self.sheetsu_api_url = "https://sheetsu.com/apis/v1.0/"
+        self.sheetsu_api_url = "https://sheetsu.com/apis/v1.0oy/"
```
Comment (andreffs18) [inline sheetsu/core/core.py]: Hey @jdrzj , can you tell me why the **oy** in the end of the url?

Comment (jdrzj) [reply sheetsu/core/core.py]: *oy* it's our way to know how users are using API. *o* is for language libs, and *y* is for python.

Comment (andreffs18) [reply sheetsu/core/core.py]: Ah, okay. Cool!

**Comment by jdrzj** [comment]
🎉

**Pull request merged**